package.json

```diff
@@ -46,13 +46,13 @@
   },
   "version": "10.0.0-rc.2",
```

Comment:
Should this be bumped too?

Reply:
Not yet, I'm going to do that with the release staging tool

Reply:
I think this is just the update to Angular v10 and CLI v10. The actual release change w/ changelog and version bump is separate (even though the peer dependency range is updated here)